public/index.html

```diff
@@ -6,6 +6,8 @@
     <body>
 	<div id="container">
 	</div> <!-- container -->
+	<div id="letter">
+	  </div>
```

Comment:
Thanks Alex - my editor sometimes does strange things when pushing and pulling code and I think I just forgot to clean this one up.